packages/components/src/components/toggle-switch/toggle-switch.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes                      from 'prop-types';
+import classNames                     from 'classnames';
+
+class ToggleSwitch extends Component {
+    render() {
+        return (
+            <Fragment>
+                <input
+                    className={classNames('dc-toggle-switch', {
+                    }, this.props.className)}
+                    id={this.props.id}
+                    type='checkbox'
+                    checked={this.props.is_enabled}
+                    onChange={this.props.handleToggle}
+                />
+                <label
+                    className={classNames('dc-toggle-switch__label', {
+                    }, this.props.classNameLabel)}
```

Comment:
@mustofa-binary i dont think you need to use classnames here. just use object literals